gamestonk_terminal/stocks/technical_analysis/ta_controller.py

```diff
@@ -133,6 +139,7 @@ def print_help(self):
     bbands      bollinger bands
     donchian    donchian channels
     kc          keltner channels
+    capm        capm values
```

Comment:
I would have this in description "capital asset pricing model"

Reply:
> I would have this in fundamental_analysis

Technically it would be technical analysis because it only looks at stock price, and not at the financial statements or health of the overall economy. But if you want it moved I can do it.

Reply:
Yea, I see what you mean. I'll let @jmaslek decide this one 😄

Reply:
Uhhh. I flipped a coin and it landed on heads so ta (for now).

Honestly I could also see this in qa. 

Having risk models in portfolio would be sweet and this would be another place to have.

Reply:
Yes, I think I prefer the qa idea. (Sorry for the coin flipping 😄 )

Reply:
Moved!